static/js/init.js

```diff
@@ -11,6 +11,7 @@ var tables = require('./modules/tables.js');
 var progress = require('./modules/progress.js');
 var errors = require('./modules/errors.js');
 var router = require('./modules/router.js');
+//var mocks = require('../../tests/mocks/mocks.js');
```

Comment:
I'm leaving these comments in place because they are useful in writing/debugging tests off of a local environment and it gets stripped out when Browserify condenses the JS into a production version.